proto/pulumi/provider.proto

```diff
@@ -323,10 +324,11 @@ message ConstructRequest {
     map<string, PropertyDependencies> inputDependencies = 11; // a map from property keys to the dependencies of the property.
     bool protect = 12;                                        // true if the resource should be marked protected.
     map<string, string> providers = 13;                       // the map of providers to use for this resource's children.
-    repeated string aliases = 14;                             // a list of additional URNs that shoud be considered the same.
+    repeated string aliasURNs = 14;                             // a list of additional URNs that shoud be considered the same.
```

Comment:
I was going to leave this and analyzers as just taking the urn strings for now, it's a lot of extra blast radius to worry about how these will interact.